src/routes/plans/[id].svelte

```diff
@@ -170,9 +170,8 @@
   }
 
   function onKeydown(event: KeyboardEvent): void {
-    const { key, metaKey } = event;
-
-    if (metaKey && key === 's') {
+    const { key, ctrlKey, metaKey } = event;
+    if ((window.navigator.platform.match('Mac') ? metaKey : ctrlKey) && key === 's') {
```

Comment:
Is there a reason `match` is used here instead of

Reply:
Oh I forgot to make it regex and case insensitive. For that case, match could be used, but for what it is currently, I agree that `includes` is better. Which do you prefer?

Reply:
ah makes sense. Will that string change? Fine with making it a regex and keeping match.

Reply:
I doubt it will. It's just me being overcautious. I'll make it regex if you don't mind.